src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -558,36 +558,6 @@ void SyncInferRequest::init_tensor(const std::size_t& port_index, const ov::ISyn
                     tensor = ov::make_tensor(model_prec, tensor_shape);
                 }
                 ov::ISyncInferRequest::set_tensor(port, tensor);
-            } else {
-                const auto& tensor_shape = tensor->get_shape();
```

Comment:
@maxnick @zhaixuejun1993 @riverlijunjie @ilya-lavrenov some time ago we met related issue with `Parameter->Result` case (see [CVS-115911](https://jira.devtools.intel.com/browse/CVS-115911)). As a solution, we proposed `Identity` insertion in order to have 2 separate tensors via some transformation, and such `Identity` may be somehow managed/ignored by plugins. I propose we should fix the issue this way, may we discuss?

Reply:
Introducing `Identity` op and using it to resolve some framework specific Parameter->Results subgraphs doesn't affect the ngraph core logic and has zero impact on what is being done in this PR. The only action that may be required from the plugin side is an optimized implementation of the future `Identity` op, but this will be handled withing the well established opset enablement process.